### ISSUE TITLE
Pages are not translated when selecting a language pack

### ISSUE THREAD

**Issue by remogatto** [CampbellSoftwareSolutions/docker-osticket#76]
When selecting Italian as system language, translations appear incomplete. In particular, pages are not translated (the landing page, for example). I suspect this depends on the fact that the content of the pages is written directly into the database during the installation phase. Is there a way to solve the issue? Thank you for this awesome docker image!

**Comment by Netzvamp** [comment]
I've tried installing osticket without docker on an regular webserver and it also doesn't translate database content when i selected another language in the setup. If you look in the i8n folder, there are only yml files for en_US, that are the files for database content. The additional .phar are only for the UI after database's inital setup.

TLDR: It happens also without docker :(
